### PR TITLE
Changes Capsaicin On-Touch Protection Requirement

### PR DESCRIPTION
**Pull request by Wisemonster**
[BUFF]

## About the PR 
Changes capsaicin's on-touch protection to require eye cover to prevent it's effect. The eye cover is the same that prevents eye surgery, and sechuds.

## Why's this needed? 
Currently, the only thing you need to protect yourself from the on-touch effect of capsaicin is any kind of mask, including the one everyone spawns with in their bag. This aims to make capsaicin more useful, by requiring more protection to prevent harm from it.

## Changelog

```changelog
(u)Wisemonster
(*)Capsaicin now requires eye covering to avoid it's On-touch effect. (For reference, if it blocks eye surgery, or it's sechuds, it'll block the on-touch effect of capsaicin.)
```

### CODE REVIEW

File: code/modules/chemistry/Reagents-FoodDrink.dm [hunk 1]
```diff
@@ -1948,16 +1948,29 @@ datum
...
-							if(prob(50))
-								M.emote("scream")
-								boutput(M, "<span class='alert'><b>Your eyes hurt!</b></span>")
-								M.take_eye_damage(1, 1)
-							M.change_eye_blurry(3)
-							M.changeStatus("stunned", 2 SECONDS)
-							M.change_misstep_chance(10)
+					if(ishuman(M))
+						var/mob/living/carbon/human/H = M
+						if((H.wear_mask?.c_flags & COVERSEYES) || (H.head?.c_flags & COVERSEYES) || H.glasses && istype(H.glasses, /obj/item/clothing/glasses/sunglasses/sechud))
```
Comment (ZeWaka) [inline code/modules/chemistry/Reagents-FoodDrink.dm:1953]: Why sechud

Comment (Wisemonster) [reply code/modules/chemistry/Reagents-FoodDrink.dm:1953]: I added sechuds as protection due to sec having access to grenades that puts out a cloud of capsaicin, (AKA, the crowd dispersal grenades.)  and HoS's hat didn't provide protection from the gas.

Comment (ZeWaka) [reply code/modules/chemistry/Reagents-FoodDrink.dm:1953]: Why do they not cover eyes

Comment (Wisemonster) [reply code/modules/chemistry/Reagents-FoodDrink.dm:1953]: If your talking about the sechuds, currently all glasses count as COVERSEYES, even if some of them don't make sense. I didn't want to have all glasses provide protection against it, however I can change it to use the COVERSEYES proc for glasses if necessary. (Like prescription glasses and eyepatches) Although, a pr to change which glasses has the COVERSEYES proc would help with this. If your talking about the HoS hat, apparently that was changed some time ago to no longer have the COVERSEYES proc.

**Pull request closed without merging**